385/Assets/Scripts/PlayerController.cs

```diff
@@ -178,8 +185,19 @@ void Update ()
                 onGround = false;
             }
         }
+        changeCam();
 	}
 
+    private void changeCam()
+    {
+        if (Input.GetKeyDown(KeyCode.C))

```

Comment:
Because we are trying to target controller support as well, we shouldn't be checking keycodes directly. Take a look at how the jumping was done, and try to replicate that functionality.